TODO.md

```
@@ -5,4 +5,5 @@
 - [] Verify Page Tree
 - [] Update Docs for new ways
 - [x] add back search with ai in empty search area
-- [] remove all different package managers, and make it bun only
\ No newline at end of file
+- [] remove all different package managers, and make it bun only
+- ai generated links are not proper
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)